README.md

```diff
@@ -25,7 +25,7 @@
 
 The Eiffel Remrem Protocol Interface is the interface for the Eiffel Remrem Semantics (https://github.com/eiffel-community/eiffel-remrem-semantics).
 
-It is provided with the purpose that anyone should be able to create their own Eiffel Semantics library to use for Remrem. Further documentation is provided at the following link: [http://ericsson.github.io/eiffel-remrem-protocol-interface/](http://ericsson.github.io/eiffel-remrem-protocol-interface/).
+It is provided with the purpose that anyone should be able to create their own Eiffel Semantics library to use for Remrem. Further documentation is provided at the following link: [http://eiffel-community.github.io/eiffel-remrem-protocol-interface/]
```

Comment:
Missing parenthesis so the full link is still visible in markdown. Add ( ) around link to make it cleaner.